 --- Init ---
Level 4
 --- Bot used 47 milliseconds ---
-----------------------------------------------------------------------------
 --- Frame #1 ---
Traceback (most recent call last):
 --- Bot used 73 milliseconds ---
ERRORED!
Response received (if any):
Traceback (most recent call last):
 --- Bot was killed. Below is the rest of its output (if any): ---
  File "../bot.py", line 14, in <module>
    moves = [strategy.decide_move(square) for square in game_map if square.owner == myID]
  File "../bot.py", line 14, in <listcomp>
    moves = [strategy.decide_move(square) for square in game_map if square.owner == myID]
  File "/home/toothless/Desktop/coveo_training/bots/practice_bot/level_4/src/strategy.py", line 18, in decide_move
    direction = self._find_border_direction(square)
  File "/home/toothless/Desktop/coveo_training/bots/practice_bot/level_4/src/strategy.py", line 22, in _find_border_direction
    foreign_squares = [square for square in game_map if square.owner != myID]
NameError: name 'game_map' is not defined

 --- End bot output ---